cpanfile

```diff
@@ -37,7 +37,7 @@ requires 'DateTime::Locale', '>= 1.22';
 requires 'Math::Random::Secure';
 requires 'Crypt::ScryptKDF';
 requires 'Locale::Maketext::Lexicon::Getcontext', '>= 0.05';
-requires 'Email::IsEmail', '>= 3.04.8';
+requires 'Email::Valid', '>= 1.202, < 2.0';
```

Comment:
`Email::Valid` exists as a Debian package: `libemail-valid-perl`. Isn't it better to use it?

Reply:
We need the module to be listed in cpanfile, so that Travis CI and Docker can build successfully. If libemail-valid-perl is installed on Debian (stable is 1.202), then cpanm won't re-installed it either way. 🙂

Reply:
I wrote this comment because you put `Email::Valid` under `# Probably not available as Debian packages` section (but that's not very important).

Thanks a lot @hangy!